doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,305 @@
+        }
+      }
+    },
+    "aioMetadata": {
+      "type": "object",
+      "description": "The AIO requirements to run this connector",
+      "properties": {
+        "aioMinVersion": {
+          "type": "string",
+          "description": "The minimal (inclusive) version that can run this connector."
+        },
+        "aioMaxVersion": {
+          "type": "string",
+          "description": "The maximal (inclusive) version that can run this connector."
+        }
+      }
+    },
+    "inboundEndpoints": {
+      "type": "array",
+      "description": "The array of inbound endpoints that this connector connects with.",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "configSchemas": {
+              "type": "object",
+              "description": "The inbound endpoint level JSON schemas for the various 'additional configuration' fields. A JSON schema must be provided for each 'additional configuration' only if that field is used by this connector.",
+              "properties": {
+                "deviceEndpointAdditionalConfigSchema": {
+                  "$ref": "#/definitions/additionalConfigurationType"
+                }
+              }
+            },
+            "assets": {
+              "type": "array",
+              "description": "The array of assets within an inbound endpoint",
+              "items": [
+                {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string"
+                    },
+                    "configSchemas": {
+                      "type": "object",
+                      "description": "The JSON schemas for the asset-level various 'additional configuration' fields. A JSON schema must be provided for each 'additional configuration' only if that field is used by this connector.",
+                      "properties": {
+                        "datasetConfigSchema": {
+                          "$ref": "#/definitions/additionalConfigurationType"
+                        },
+                        "dataPointConfigSchema": {
+                          "$ref": "#/definitions/additionalConfigurationType"
+                        },
+                        "eventConfigSchema": {
+                          "$ref": "#/definitions/additionalConfigurationType"
+                        },
+                        "streamConfigSchema": {
+                          "$ref": "#/definitions/additionalConfigurationType"
+                        },
+                        "managementGroupConfigSchema": {
+                          "$ref": "#/definitions/additionalConfigurationType"
+                        },
+                        "processControlConfigSchema": {
+                          "$ref": "#/definitions/additionalConfigurationType"
+                        }
+                      }
```

Comment:
is this real?

Reply:
this comment was referring to processControlSchema, which is not part of ADR.

You have now removed it. Thanks.